packages/react/src/components/Modal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Modal, ModalSize } from ":/components/Modal/index";
+import { useCunningham } from ":/components/Provider";
+import { Button } from ":/components/Button";
+import { DecisionModalProps } from ":/components/Modal/ModalProvider";
+
+export type ConfirmationModalProps = DecisionModalProps;
```

Comment:
I don't see why a DeleteConfirmation or a ConfirmationModal cannot use messageType.
we could have: 
:information_source: You'll accept this "basic choice"
:warning: You'll accept this "be careful choice"
:exclamation: You'll accept this "dangerous  choice"

Reply:
It's all about icons, and number of buttons and `onDecide` value, the components remains simple IMO